src/components/data-exploration/heatmap/HeatmapPlot.jsx

```diff
@@ -172,6 +173,8 @@ const HeatmapPlot = (props) => {
   ]);
 
   useConditionalEffect(() => {
+    // TODO: we could avoid having to reload the marker heatmap all the time same way
+    // we do for embeddings & Gene Expression
```

Comment:
is this TODO still relevant?

Reply:
See previous comment. The heatmap related data always triggers a new work request. Not urgent (not necessary either), maybe in the future.